rest/route/host_agent.go

```diff
@@ -1238,22 +1238,14 @@ func (h *hostAgentEndTask) Run(ctx context.Context) gimlet.Responder {
 		endTaskResp.ShouldExit = true
 	}
 
-	// we should disable hosts and prevent them from performing
-	// more work if they appear to be in a bad state
-	// (e.g. encountered 5 consecutive system failures)
+	// Disable hosts and prevent them from performing more work if they have
+	// system failed many tasks in a row.
```

Comment:
nit: many -> consecutiveSystemFailureThreshold

Reply:
I think the comment is fine as-is, you can see the `consecutiveSystemFailureThreshold` immediately below and the comment is only there to state the general reasoning of why this special behavior is here, not restate what the code is doing.